docs/openapi/components/schemas/common/LEIentity.yml

```diff
@@ -167,7 +168,8 @@ example: |-
     "headquartersAddress": {
       "type": ["LEIaddress"],
       "language": "en_BORK",
```

Comment:
[`en_BORK`](https://localizely.com/language-code/en/) doesn't exist.

Reply:
fixed. And there was another case of using `_` whereas `-` is correct